doc/UseCases/uc-cp_x-road_configuration_proxy_use_case_model_1.2_Y-883-5.md

```diff
@@ -0,0 +1,1032 @@
+    proxy.
+
+References
+----------
+
+1.  []{#Ref_PKCS11 .anchor}\[PKCS11\] PKCS \#11 Cryptographic Token
+    Interface Base Specification Version 2.40. Function return values.
+    <http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/csprd01/pkcs11-base-v2.40-csprd01.html#_Toc372627249>
+
+2.  []{#Ref_PR-GCONF .anchor}\[PR-GCONF\] X-Road: Protocol for
+    Downloading Configuration. Document ID: PR-GCONF.
+
+3.  \[[]{#Ref_UG-CP .anchor}UG-CP\] X-Road: Configuration Proxy Manual.
+    Document ID: UG-CP.
+
+4.  []{#Ref_UC-GCONF .anchor}\[UC-GCONF\] X-Road: Use Case Model for
+    Global Configuration Distribution. Document ID: UC-GCONF.
+
+Overview
+========
+
+The configuration proxy acts as an intermediary between X-Road servers
+in the matters of global configuration exchange.
+
+The goal of the configuration proxy is to download an X-Road global
+configuration from a provided configuration source and further
+distribute it in a secure way.
+
+The configuration proxy can be configured to mediate several global
+configurations (from multiple configuration sources).
+
+Use Case Model
+==============
+
+Actors
+------
+
+The use case model for the X-Road configuration proxy includes the
+following actors.
+
+-   **CP administrator** – Configuration proxy administrator – a person
+    responsible for managing the configuration proxy.
+
+-   **Configuration client** – System acting as a configuration client.
+    Can either be a security server or another configuration proxy.
+
+-   **Configuration source** – a component (HTTP server) that
+    distributes global configuration and is managed by the central
+    server or another configuration proxy.
+
+Relationships between the actors, the system and the use cases are
+described in Figure 1.
+
+![](img/use_case_diagram_for_the_conf_proxy.PNG)
+
+Figure 1. Use case diagram for the configuration proxy
+
+UC CP\_01: View Proxy Settings
+------------------------------
+
+**System**: []{#magicparlabel-7226 .anchor}Configuration proxy
+
+**Level**: User task
+
+**Component**: Configuration proxy
+
+**Actor**: CP administrator
+
+**Brief Description**: CP administrator views the settings of the
+configured configuration proxy instances.
+
+**Preconditions**: -
+
+**Postconditions**: The settings are displayed to CP administrator.
+
+**Trigger**: -
+
+**Main Success Scenario**:
+
+1.  CP administrator selects to view all proxy instance settings.
+
+2.  System displays list of proxy instances. For each instance, the
+    following information is displayed:
+
+    -   trusted anchor;
+
+    -   configuration download URL;
+
+    -   active signing key identifier and respective certificate;
+
+    -   list of inactive signing key identifiers and respective
+        certificates;
+
+    -   configuration validity interval.
+
+**Extensions**:
+
+1a. CP administrator selects to view the settings for a specific proxy
+instance.
+
+> 1a.1. System displays information from step 2 for the requested
```

Comment:
Markdown quotation looks misleading in Github. For example uc-mess looks a lot better:
1a. Request was sent using a method other than POST.
  - 1a.1. System sends a SOAP Fault message with fault string “Must use POST request method instead of X” (where “X” is the used method) to the Client IS. The use case terminates.